src/main/java/com/hillel/items_exchange/controller/ImageController.java

```diff
@@ -78,11 +80,15 @@
             @ApiResponse(code = 406, message = "NOT ACCEPTABLE"),
             @ApiResponse(code = 415, message = "UNSUPPORTED MEDIA TYPE")})
     public ResponseEntity<String> saveImages(@PathVariable("product_id")
-                                                     @PositiveOrZero(message = "{invalid.id}") long productId,
-                                                 @RequestParam(value = "files") @Size(min = 1, max = 10) List<MultipartFile> images) {
+                                             @PositiveOrZero(message = "{invalid.id}") long productId,
+                                             @RequestParam(value = "files") @Size(min = 1, max = 10) List<MultipartFile> images)
+            throws ElementsNumberExceedException {
 
```

Comment:
Why didn't you add a constant (something like MAX_IMAGES_AMOUNT) for using it in all place?

Reply:
From mine point of view it isn't so critical. I used such value only two time here and each time with different meaning: inside Side annotation as limitation for a request and inside if block as limitation for total quantity of children. 
Let's wait for other guys opinion about extracting it into a new variable or property.

Reply:
Maybe should to extract this variable to global variables? This solution of course not allow to use it in `@Size` annotation, but this will allow us in case we want to change this variable - change it in application.properties and that's all. P.S. you can see how it is realized in CGE-118. But be carefull, I find some unwanted behavior at display message error, I prupose solution from day to day at new PR

Reply:
Our application grows :) and we have some constant values used at different places like max children amount or max images one, so I'm agreed each of them should be gotten from one place. But moving them to the application.properties doesn't allow us to use them with `@Size`, so in my opinion the place to move should be one for all usages from or do nothing with it :)

Reply:
OK. I'll done it only for the method body.